compiler/include/driver.h

```diff
@@ -106,6 +106,8 @@ extern const char* CHPL_WIDE_POINTERS;
 extern const char* CHPL_LLVM;
 extern const char* CHPL_AUX_FILESYS;
 extern const char* CHPL_UNWIND;
+extern const char* CHPL_RUNTIME_SUBDIR;
+extern const char* CHPL_LAUNCHER_SUBDIR;
```

Comment:
Maybe I'm just missing something obvious, but where is `CHPL_LAUNCHER_SUBDIR` actually used?

Reply:
No, it's not, but one day I'd like the included clang to build the launcher (vs running Make etc). I was "planning for the future". I could remove it if that's important to you.